### ISSUE TITLE
Healthcare Violations

### ISSUE THREAD

**Issue by tsp2123** [jsoma/data-studio-projects#169]
# Pitch

## Summary

I found a dataset of industry violations tracked by Watchdog group Good Jobs First. I'm looking at the Healthcare Industry to ask the following questions:

Which company has the most violations?

Which year did these violations occur?

Do some US States have more violations than others?

What was the primary reason for these violation?

## Details

**Possible headline(s):** 

**Data set(s):** 
https://violationtracker.goodjobsfirst.org/prog.php?major_industry_sum=healthcare+services

**Code repository:** 
	
**Possible problems/fears/questions:** 

## Work so far

Here we can see that certain companies have excessively more violations than others:

![screen shot 2018-07-17 at 1 34 50 am](https://user-images.githubusercontent.com/39416100/42798439-be0f911c-8961-11e8-8f42-0c7d0fecd6a6.png)

The Top Two seem to be Kaiser Permanente and American Medical Response. I'm still trying to work my data, but the following graph shows me trying count how many violations per year. It's not working out right now with the dataset sorting itself for whatever reason.

![screen shot 2018-07-17 at 1 35 01 am](https://user-images.githubusercontent.com/39416100/42798494-f3dd203e-8961-11e8-8349-c516f1b05e1f.png)

## Checklist

This checklist must be completed before you submit your draft.

- [ x] I have already spent time with my data set, opening it, exploring it, etc
- [ ] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [ x] My issue links to my data set(s)
- [ ] My issue links to my code repository
- [x ] My issue explains what I'd like to explore in the data set
- [x ] My issue includes images - either inspiration or what I've done so far

**Comment by playfairbot** [comment]
Hello! I'm a little robot, let's take a peek.

Please post your **first revision!** It should be posted by **Thursday at midnight**. More details available [here](/jsoma/data-studio-projects).

You need some feedback, let me summon @SimoneLuc, @SiruiZhu, @jessimckenzi for you

It looks like we need to **fix up [your pitch](https://github.com/jsoma/data-studio-projects/issues/169)** a little bit! Edit it by clicking the pencil in the top right-hand corner. It requires:

* Sample headlines
* A link to your code repository

**Comment by jessimckenzi** [comment]
Interesting data! I'm unclear what's being counted here, but 13 violations doesn't seem like that many more than 5, without context. As for the styles, the colors are really aggressive, and it appears as though you're trying to draw attention to the red ones, even though it doesn't look like those are special in any way. And what's the 3 in the second graph? These are (apparently) very small numbers you're working with—why are they notable?

**Comment by SiruiZhu** [comment]
I like this idea!! My thoughts would be:

- Put the top at the top of the horizontal bar chart, also there are too many colors here, actually you could maybe unify them as one color and highlighted the top 1?

**Comment by tsp2123** [comment]
# Update
Sorry about the late post. Here's some updates
## Your project content: images/words/etc
	
![screen shot 2018-07-22 at 11 15 19 pm](https://user-images.githubusercontent.com/39416100/43055615-9b22354e-8e05-11e8-9988-b657bbee72a7.png)

![screen shot 2018-07-22 at 11 15 29 pm](https://user-images.githubusercontent.com/39416100/43055617-9c730bf8-8e05-11e8-80ba-aca87c2ac2f0.png)

![screen shot 2018-07-22 at 11 15 37 pm](https://user-images.githubusercontent.com/39416100/43055619-9daea400-8e05-11e8-9304-b4b9d6c95a54.png)

![screen shot 2018-07-22 at 11 15 49 pm](https://user-images.githubusercontent.com/39416100/43055622-9efb4d90-8e05-11e8-8e4a-b915a968c780.png)

  
## Any changes in direction or topic?
  
I realized my initial analysis wasn't as in depth and I'm still in an exploratory phase. My next steps are to look deeper into the biggest violators by company 

	
## Problems/Questions
  
I am having an initial problem when I want to make a bar graph of violations by year. For whatever reason, the graph decides to sort itself whereas I need the X axis to be chronological 
	
## Checklist
	
- [ ] I have included my visuals
- [ ] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by tsp2123** [comment]
# Update
	
## Your project content: images/words/etc

Here are some revisions to my old graphs.

![healthcare_violations_years](https://user-images.githubusercontent.com/39416100/43273724-2e00ea82-90cb-11e8-8f76-4e2c3b164f4d.png)

![cumalative_violations](https://user-images.githubusercontent.com/39416100/43276514-e5a363fc-90d2-11e8-9230-a8fc1dadc37f.png)


## Any changes in direction or topic?
  
I've changed some of the aesthetics of this graph. I'm still getting a hang of illustrator and trying to fit things into an art board template
	
## Problems/Questions
  
I've been trying to find ways to dig deeper into the companies but I'm not confident in the dataset's collection. For example the info links dont like to much more information that what's already present in the dataset and there are no links to any PACER case files so while you can tell the number of violations and the addresses at which these violations happen you cant tell much else about it : ( 

The other question I've reached out for an explanation—but related to the lack of data clarity—I'm not sure how Wage and Hour violations are different from Labour Violations and whether that's even a worthy distinction for my audience. If I consider them together, it would change my dataset greatly, and I'm not sure whether that's worth doing. 
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by sarahslo** [comment]
if you have a chart that has a scale you don't also need numbers on it. and watch out for repeating something, 'violations' should be a label once, not repeated.

i find this dataset problematic. partly because it's from a watchdog group, and that makes me wonder what is their agenda. but partly because the numbers are so small.

the fact that certain companies have more violations than others, some are bigger than others and handle more clients.

these data are not strong enough to make charts out of. to be honest, i would scrap the data entirely. once you harden something into a chart it makes it appear to be fact and in this case, its misleading. your instinct to not be confident about the numbers, go with it.

**Comment by ella24** [comment]
I was confused about the graph Healthcare violations by year'. The subtitle mentioned that they have been increasing every year, however, we could see some periods where they started to lower, especially after 2015.
I approve that you changed that subtitle, but I encouraged you to use annotations to explain what was happening in those periods. Yes, that is going to be another research, but a quite revealing one, since your data is not that broad (specify the source in the graph).
It is great that you decided to use just one color for the cumulative violations, the first version was quite confusing. However, I do not think that the bar graph is the best one to use here. Maybe pie? I know Soma does not like them, but in this topic, I think they will communicate faster and punctually. 
To compare companies, it would be nice to have more information about their available resources and size. This way we can compare more accurately.

**Comment by hakantan** [comment]
I would love it to see what a "False Claims Act" actually is. 

On one of your final charts the term "violations" is mentioned 12 times. I would use it just in the headline and as an axis-label. Instead of rotating the years (x-axis), maybe it would make sense to have fewer years on there?

Also, since we're talking about 140 violations in 2016, my immediate questions are:

1. Why is this number so low?
2. What is "low"? (How does  it compare with at least one different industry.)

Meaning: I would need the numbers to be explained.

**Comment by tsp2123** [comment]
# Final

## Project visuals/text

![healthcare_violations_line_final](https://user-images.githubusercontent.com/39416100/43371797-10b757ec-9367-11e8-8648-267bb5aa86c0.png)

![worst_healthcare_provider_2016_final](https://user-images.githubusercontent.com/39416100/43371803-2470befe-9367-11e8-9210-04e3bb14d5b3.png)


![top_5_2016_final](https://user-images.githubusercontent.com/39416100/43371806-298db7de-9367-11e8-84d1-0d8c79ab750b.png)

![fines_2016_final_c](https://user-images.githubusercontent.com/39416100/43371809-3253c0c0-9367-11e8-99e8-2375928e3a9b.png)


















Here are my updated visuals! And here's the link to what I put in html. (For whatever reasons these aren't uploading but hopefully the hyperlink works and you can view them there)





## Details

**Headline:** 
Don't Put Grandma in a Nursing Home

**Published website version:**

https://tsp2123.github.io/projects.io/Project/html%20project/index.html

**Code repository:**
https://github.com/tsp2123/data-studio/tree/master/Project_2
**Final data set(s):** 
''''
## What did you find to be the most difficult part of this project?

This dataset was difficult to produce a data story out of—I think it's one of those sets that is useful for exploratory analysis, for example, the fact that there is a shit ton of issues with nursing homes seems to be replicated in this dataset and reflects the more anecdotal news of nursing home's being rife with violations, but the dataset doesn't take into consideration other important factors about the business involved such as the size of the business, its market share, etc. So as Sarah pointed out it isn't conducive to the best results and may lead to certain outcomes that aren't entirely reflective of the situation. 

## Are you satisfied with what you produced? Is there anything you would like to change or improve?

I'm not to keen on this project. Again, it's really just a test project for me querying datasets, but other than that this isn't really for publication. It's simply a test. 

## Checklist

- [ ] I have included my visuals
- [ ] I have posted my project to my project website
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by christina10211** [comment]
Hey! Congratulations on completing your project!  I took a look at your website and I would suggest to adjust the scale a little bit. I figured that you are using the size A for three of your graphs, and I feel like this might not be the perfect size for your charts since I can't read the labelling very clearly, and the font size would better be standardized to create a coherent look. 

A minor suggestion on your first graph: maybe you can make your x-axis tick size smaller :)
